src/spx_php.c

```diff
@@ -201,14 +207,14 @@ static void global_hook_execute_internal(
 
 static zend_op_array * global_hook_zend_compile_file(zend_file_handle * file_handle, int type TSRMLS_DC);
 static zend_op_array * global_hook_zend_compile_string(
-#if PHP_API_VERSION >= 20200930
+#if ZEND_MODULE_API_NO >= 20200930
     zend_string * source_string,
     const
 #else
     zval * source_string,
 #endif
     char * filename
-#if PHP_API_VERSION >= 20210903
+#if ZEND_MODULE_API_NO >= 20210903
     , zend_compile_position position
 #endif
         TSRMLS_DC
```

Comment:
Do you really need the `TSRMLS_*` parameters? These are no-ops as of PHP 7.0.0.

Reply:
Yes SPX still supports PHP 5.6